src/test/java/com/wikia/webdriver/elements/mercury/components/Head.java

```diff
@@ -0,0 +1,25 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Head {
+
+  private final WebDriver driver;
+
+  @FindBy(css = "head title")
+  private WebElement documentTitle;
+
+  public Head(WebDriver driver) {
+    this.driver = driver;
+    PageFactory.initElements(driver, this);
+  }
+
+  public String getDocumentTitle() {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
```

Comment:
I haven't thought about that, however I would like to get the "raw" version of HTML title in case we have some errors - for instance trailing whitespaces.

I didn't extend WikiBasePageObject none of other components extends it. I had the impression that compoments should be as small and independent from other parts of code as possible. If you feel it's a problem I can change that however could you create guidelines when to create what? Current repo structure is very hard to follow and searching for up-to-date patterns isn't the easiest thing to do.